src/app/shared-links/Components/dialogs/QRCodeDialog.tsx

```diff
@@ -91,7 +96,11 @@ export const QRCodeDialog: FC<QRCodeDialogProps> = ({
             </Grid>
             <Grid item textAlign="center">
               <Button
-                sx={{ maxWidth: '240px' }}
+                sx={{
```

Comment:
perhaps you could assign the style to a variable and reuse it

Reply:
Yes, it's a good idea